ui/component/markdownLink/view.jsx

```diff
@@ -107,23 +121,40 @@ function MarkdownLink(props: Props) {
       />
     );
   } else if (!simpleLinks && ((protocol && protocol[0] === 'lbry:' && isURIValid(decodedUri)) || lbryUrlFromLink)) {
-    element = (
-      <ClaimLink
-        uri={lbryUrlFromLink || decodedUri}
-        autoEmbed={embed}
-        parentCommentId={parentCommentId}
-        isMarkdownPost
-        allowPreview={allowPreview}
-      >
-        {children}
-      </ClaimLink>
-    );
+    if (isComment && isMention && isChannel && setUserMention) {
+      element = (
+        <Menu>
+          <MenuButton className="menu__button" onClick={(e) => e.stopPropagation()}>
+            <ChannelTitle uri={decodedUri} fallback={children} isComment />
+          </MenuButton>
+
+          <CommentMenuList
+            authorName={children}
+            uri={lbryUrlFromLink || decodedUri}
+            authorUri={lbryUrlFromLink || decodedUri}
+            commentIsMine={isMe(activeChannelClaim && activeChannelClaim.short_url, lbryUrlFromLink || decodedUri)}
+            isLiveComment
+          />
+        </Menu>
+      );
+    } else {
+      element = (
+        <ClaimLink
+          uri={lbryUrlFromLink || decodedUri}
+          autoEmbed={embed}
+          parentCommentId={parentCommentId}
+          // isMarkdownPost={isMarkdownPost}
```

Comment:
The removal of `isMarkdownPost` breaks embeds in Posts
(e.g. None of the embeds in http://localhost:9090/@dev-test:8/test-embed:b is playable.)